moped-editor/src/utils/taskOrderComponentHelpers.js

```diff
@@ -5,7 +5,7 @@ import { TextField } from "@mui/material";
  * Socrata Endpoint
  */
 export const SOCRATA_ENDPOINT =
-  "https://data.austintexas.gov/resource/ggav-ufvc.json?$limit=9999";
+  "https://data.austintexas.gov/resource/ggav-ufvc.json?$limit=99999";
```

Comment:
I was so hoping to find something in the API documentation that allowed for a "don't limit this query at all" functionality. 🫰🏻No Dice, of course.